spring-cloud-stream/src/test/java/org/springframework/cloud/stream/function/ProcessorToFunctionsSupportTests.java

```diff
@@ -38,30 +36,24 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Import;
 import org.springframework.integration.dsl.IntegrationFlow;
-import org.springframework.integration.dsl.IntegrationFlowBuilder;
 import org.springframework.integration.dsl.IntegrationFlows;
-import org.springframework.lang.Nullable;
 import org.springframework.messaging.MessageChannel;
 import org.springframework.messaging.support.GenericMessage;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
- *
  * @author Oleg Zhurakousky
- *
+ * @author David Turanski
  * @since 2.1
- *
  */
 public class ProcessorToFunctionsSupportTests {
 
 	@Test
 	public void testPathThrough() {
-		ApplicationContext context =
-				new SpringApplicationBuilder(
-						TestChannelBinderConfiguration.getCompleteConfiguration(FunctionsConfiguration.class))
-						.web(WebApplicationType.NONE)
-						.run("--spring.jmx.enabled=false");
+		ApplicationContext context = new SpringApplicationBuilder(
```

Comment:
@dturanski , since you already here, consider, please, to `close()` all the application contexts in the end of these tests.